easybuild/easyconfigs/p/PSI4/PSI4-1.3.2-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'EB_PSI'
+
+name = 'PSI4'
+version = '1.3.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.psicode.org/'
+description = """PSI4 is an open-source suite of ab initio quantum chemistry programs designed for
+ efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform
+ computations with more than 2500 basis functions running serially or in parallel."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'usempi': True}
+
+github_account = 'psi4'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['PSI4-%(version)s_fix_cmake_release.patch']
+checksums = [
+    'ed76c67803b6420f35f57a6dd31c47108b9145b8c9fced5c94cdc179f6b5fbf3',  # v1.3.2.tar.gz
+    'c399b6a90e587976f7ec98a3f05c3b62525d015639f8352004b5bd4bea696eda',  # PSI4-1.3.2_fix_cmake_release.patch
+]
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    # Perl needed for the ctest test suite during the installation, not for the pytest test suite after the installation
+    # ('Perl', '5.30.0'),
+]
+
+dependencies = [
```

Comment:
Would it be worth it to add a dependency on `Libint`? Currently it is compiling the bundled library but we already have easyconfigs for it.

Reply:
But we don't have one for 2019b.... so due to lack of time right now, I'll leave that out.

Reply:
They are still using Libint v1.2.1 and we're at 2.5/2.6 (currently no ec for Libint at intel/2019b level) so due to lack of time i won't try at the moment.